docs/reference/platforms.rst

```diff
@@ -38,29 +38,30 @@ invoking::
 The GTK+ backend is reasonably well developed, but currently has some known issues
 with widget layout.
 
-.. _toga-gtk: http://github.com/pybee/toga-gtk
+.. _toga_gtk: https://github.com/pybee/toga/tree/master/src/gtk
 
-Win32
+Winforms
 ~~~~~
 
-The backend for Windows is named `toga-win32`_. It supports Windows XP or
-later. It is installed automatically on Windows machines (machines that report
-``sys.platform == 'win32'``), or can be manually installed by invoking::
+The backend for Windows is named `toga-winforms`_. It supports Windows XP or
+later with .NET installed. It is installed automatically on Windows machines
+(machines that report ``sys.platform == 'win32'``), or can be manually
+installed by invoking::
 
-    $ pip install toga[win32]
+    $ pip install toga[winforms]
 
 The Windows backend is currently proof-of-concept only. Most widgets have not been
 implemented.
 
-.. _toga-win32: http://github.com/pybee/toga-win32
+.. _toga-winforms: https://github.com/pybee/toga/tree/master/src/winforms
 
 Mobile platforms
 ----------------
 
 iOS
 ~~~
 
-The backend for iOS is named `toga-iOS`_. It supports iOS 6 or later. It
+The backend for iOS is named `toga_iOS`_. It supports iOS 6 or later. It
```

Comment:
Also here, why changing the `toga_iOS` to `toga-iOS`?

Reply:
Both names are really valid; one is the module name, and one is the package name registered on PyPI - but the module name will also work as a package name.